scripts/generate-full-corvid-types.js

```diff
@@ -1,48 +1,13 @@
 #!/usr/bin/env node
-const tmp = require("tmp");
 const fs = require("fs-extra");
-const path = require("path");
-const ts = require("typescript");
 const without_ = require("lodash/without");
+const createTsProgram = require("./createTypescriptProgram").createTsProgram;
```

Comment:
See my previous comment - that way this line can change to
```suggestion
const createTsProgram = require("./createTypescriptProgram");
```